full-moon/tests/common/mod.rs

```diff
@@ -0,0 +1,18 @@
+use std::fs;
+use std::path::Path;
+
+fn with_insta_settings(path: &Path, f: impl FnOnce()) {
+    let mut settings = insta::Settings::clone_current();
+    settings.set_prepend_module_to_snapshot(false);
+    settings.set_snapshot_path(path);
+    settings.set_input_file(path);
+    settings.bind(|| f())
```

Comment:
Does this not work with `settings.bind(f())`?

Reply:
It does. A previous iteration had been passing `path` to it, which needed the lambda. Fixed.